types/connector_lock.yaml

```diff
@@ -0,0 +1,37 @@
+description: EV and EVSE board support types
+types:
+  ConnectorLockError:
+    description: >-
+      Errors for Connector Lock
+
+      These following error types give more details about connector lock errors.
+      You should consider setting a MREC_1_ConnectorLockFailure to indicated a generic lock failure with an MREC error code as well
+      as one of the more detailed but custom error codes to specify the reason:
+
+      ConnectorLockCapNotCharged: The capacitor for connector lock motor failed to charge within expected time
+      ConnectorLockUnexpectedOpen: The connector lock feedback does return open while it should be close
```

Comment:
just `returns` and "while it should be closed".

Applies also for the line below

Reply:
fixed